src/holodeck/environments.py

```diff
@@ -601,20 +601,6 @@ def set_control_scheme(self, agent_name, control_scheme):
         else:
             self.agents[agent_name].set_control_scheme(control_scheme)
 
-    def set_sensor_enabled(self, agent_name, sensor_name, enabled):
-        """Enable or disable an agent's sensor.
-
-        Args:
-            agent_name (:obj:`str`): The name of the agent whose sensor will be switched
-            sensor_name (:obj:`str`): The name of the sensor to be switched
-            enabled (:obj:`bool`): Boolean representing whether to enable or disable the sensor
-        """
-        if agent_name not in self._sensor_map:
-            print("No such agent %s" % agent_name)
-        else:
-            command_to_send = SetSensorEnabledCommand(agent_name, sensor_name, enabled)
```

Comment:
Do you need to delete the `SetSensorEnabledCommand` class?

Reply:
Yes--good catch.